src/utils/siteColorUtils.js

```diff
@@ -48,6 +48,10 @@ const createPageStyleSheet = (repoName, primaryColor, secondaryColor) => {
     `#key-highlights .col { background-color: ${primaryColor} !important;}`,
     0
   )
+  customStyleSheet.insertRule(
+    `#key-highlights .col:hover { background-color: ${primaryColor} !important;}`,
+    0
+  )
```

Comment:
could i double check on why we need this? this seems to be because our preview is stateless right?

Reply:
ideally, this should be the job of a theme provider at the root of our preview class - this might be a premature optimization for now so i'm fine with it as-is but wdyt about filing an issue for this?

Reply:
Yeah I think this would be useful, but out of scope and not as urgent  - I've filed an issue for this! #915